.ci/wercker.sh

```diff
@@ -276,6 +276,32 @@ no-exception-alot-of-projects)
   rm -rf contribution
   ;;
 
+imports-guava)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  checkout_from https://github.com/checkstyle/contribution.git
+  cd .ci-temp/contribution/checkstyle-tester
+  groovy ./launch.groovy --checkstyleVersion ${CS_POM_VERSION} \
+    --listOfProjects checks-import-order/projects-to-test-imports-guava.properties \
+    --config checks-import-order/checks-imports-error-guava.xml
+  cd ../../
```

Comment:
we do not need just execution .... it is not no-exception validation.
We need to make sure that there are no errors in report.
please grep `.ci-temp/contribution/checkstyle-tester/reports/guava/site/index.html` to make sure report is empty

Reply:
done